config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -51,7 +51,7 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210902-e4567b8e9c-go-canary
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210902-e4567b8-go-canary
```

Comment:
This is because I manually pushed the shorter tagged image over (ref: https://github.com/kubernetes/k8s.io/issues/1523#issuecomment-916504977).  Future bumps are going to go back to the longer image tag

Reply:
This is also proof that https://github.com/kubernetes/test-infra/pull/23532 is working as expected, yay 🎉